docs/source/release_notes.rst

```diff
@@ -8,35 +8,15 @@ Future Release
   * Enhancements
   * Fixes
   * Changes
-  * Documentation Changes
-  * Testing Changes
-        * Add additional reviewers to minimum and latest dependency checkers (:pr:`1070`, :pr:`1073`, :pr:`1077`)
-
-    Thanks to the following people for contributing to this release:
-    :user:`gsheni`
-
-v0.5.1 Jul 22, 2021
-===================
-    * Enhancements
-        * Store inferred datetime format on Datetime logical type instance (:pr:`1025`)
-        * Add support for automatically inferring the ``EmailAddress`` logical type (:pr:`1047`)
-        * Add feature origin attribute to schema (:pr:`1056`)
-        * Add ability to calculate outliers and the statistical info required for box and whisker plots to ``WoodworkColumnAccessor`` (:pr:`1048`)
-        * Add ability to change config settings in a with block with ``ww.config.with_options`` (:pr:`1062`)
-    * Fixes
-        * Raises warning and removes tags when user adds a column with index tags to DataFrame (:pr:`1035`)
-    * Changes
-        * Entirely null columns are now inferred as the Unknown logical type (:pr:`1043`)
-        * Add helper functions that check for whether an object is a koalas/dask series or dataframe (:pr:`1055`)
-        * ``TableAccessor.select`` method will now maintain dataframe column ordering in TableSchema columns (:pr:`1052`)
         * The criteria for categorical type inference have changed (:pr:`1065`)
         * The meaning of both the ``categorical_threshold`` and
```

Comment:
Can we make this just 1 line? Or is there a good reason to split it?

Reply:
Not sure.  Those are part of the bullet points that went into the 0.5.1 release notes.

Reply:
Alright, we can keep it as is